frame/support/test/tests/pallet.rs

```diff
@@ -17,6 +17,7 @@
 
 use frame_support::{
 	dispatch::{Parameter, UnfilteredDispatchable},
+	scale_info,
```

Comment:
Huh, does this pallet not import `scale_info`? It's a bit surprising to have it imported from `frame_support`...

Reply:
Yeah not sure why I've done that. I'll change it.